Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionQENS.py

```diff
@@ -271,7 +271,7 @@ def _warn_negative_integral(self, ws, message):
         for item in mtd[tmp_int]:
             for index in range(item.getNumberHistograms()):
                 if item.readY(index)[0] <= 0:
-                    raise RuntimeError('Negative or 0 integral in spectrum #{0} {1}'.format(index,message))
+                    self.log().warning('Negative or 0 integral in spectrum #{0} {1}'.format(index,message))
```

Comment:
This warning could be output many times.

Reply:
That's expected. It is output for every spectra that gets negative.

Reply:
OK